<launch>


	<node name="kcf" pkg="track_pkg" type="trackxy" output="screen">						
		<param name="color_image" value="/camera/color/image_raw"/>
		<param name="depth_image" value="/camera/aligned_depth_to_color/image_raw"/>
		

	</node>


</launch>
